### PR TITLE
Add import export step to end to end tests

### PR DESCRIPTION
**Pull request by li-roy**

### CODE REVIEW

File: test/test_jit.py [hunk 1]
```diff
@@ -6352,7 +6352,9 @@ class Config:
         premise = torch.LongTensor(48, 128).random_(0, 100)
         hypothesis = torch.LongTensor(24, 128).random_(0, 100)
 
-        self.checkTrace(SNLIClassifier(Config()), (premise, hypothesis), inputs_require_grads=False)
+        # TODO: toggle export_import once LSTM isn't a python op
```
Comment (zdevito) [inline test/test_jit.py]: It should not be a python op right now, is this still the case? (changes landed a few days ago)

**Review by zdevito — APPROVED** [review]
Cool -- so what end to end test still need fixes to work?

**Comment by li-roy** [comment]
@zdevito test_snli still has a couple of python ops that we can't get rid of as is.

**Pull request closed without merging**